src/h5cpp/contrib/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+set(CONTRIB_SOURCES)
+set(CONTRIB_HEADERS)
+
+if(H5CPP_ENABLE_STL)
+    add_subdirectory(stl)
+endif()
+
+if(H5CPP_ENABLE_NEXUS)
+    add_subdirectory(nexus)
+endif()
+
+set(h5cpp_sources ${h5cpp_sources} ${CONTRIB_SOURCES} PARENT_SCOPE)
+set(h5cpp_headers ${h5cpp_headers} ${CONTRIB_HEADERS} PARENT_SCOPE)
```

Comment:
The clang compiler in macOS built reports a lot of 
```
warning: no newline at end of file [-Wnewline-eof]
```
It would be good to execute
```bash
sed -i -e '$a\'  *.hpp *.txt
```
on your new files.
```